api/api.go

```diff
@@ -40,6 +40,10 @@ type Endpoint struct {
 	Method []string
 	// HTTP Path e.g /greeter. Expect POSIX regex
 	Path []string
+	// Body destination
+	// "*" or "" - top level message value
+	// "string" - inner message value
+	Body string
```

Comment:
Why do we need body?

Reply:
rpc endpoint need to say how to use body, it can be used as full message or as it part in case of nested message.
for ex:
```
message AccountUpdateReq {
  Account account = 2;
  ....
}

message Account {
  string id = 1;
  string email = 2;
  string birthday = 3;
  ...
}
```

so you can write in proto
```
rpc AccountUpdate(AccountUpdateReq) returns (AccountUpdateRsp) {
  option (google.api.http) = { put: "/api/account/{account.id}"; body: "account" };
}
```

in this case id in struct will be filled via path param, and other fields from body